journal/fs.go

```diff
@@ -109,9 +109,17 @@ func (f *fsJournal) rollJournalFile() error {
 		_ = f.fi.Close()
 	}
 
-	nfi, err := os.Create(filepath.Join(f.dir, fmt.Sprintf("lotus-journal-%s.ndjson", build.Clock.Now().Format(RFC3339nocolon))))
+	// check if journal file exists
+	if _, err := os.Stat(filepath.Join(f.dir, "lotus-journal.json")); err == nil {
```

Comment:
Technically, this could be a directory, but even if it is we want to rename it, so it's fine that we don't use the stat output.